WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -4,13 +4,19 @@ def competition_message_for_user(competition, user)
     messages = []
     registration = competition.registrations.find_by_user_id(user.id)
     if registration
-      messages << "You are " + (registration.accepted? ? "registered." : "currently on the waiting list.")
+      # i18n-tasks-use t('competitions.messages.registered')
+      # i18n-tasks-use t('competitions.messages.waiting_list')
+      messages << t('competitions.messages.registration_tooltip', message: registration.accepted? ? t('competitions.messages.registered') : t('competitions.messages.waiting_list'))
```

Comment:
For a reason I can't explain `i18n-tasks` is not able to detect these translations as parameters...
Which feels weird to me because the doc claims it's using Ruby's AST to detect the `t()` calls, so it should definitely visit this kind of nodes...

You're right about splitting the strings! I'll do that for all the locations where you commented, it will actually enable me to do a slightly better French translation :)